features.bzl

```diff
@@ -49,6 +49,16 @@ _flags = struct(
     ),
 )
 
+_java = struct(
+    # Whether JavaInfo has the module_flags_info field. Added in
+    # https://github.com/bazelbuild/bazel/commit/2217b13cae4110b0e2b8fe6a283a9b6dfbf150e8
+    java_info_module_flags_info = ge("6.0.0-pre.20220517.1"),
```

Comment:
Can you check for the presence of this method via `hasattr(my_info, "module_flags_info")`? If so, that's the preferred way to detect this feature.

Reply:
Sure, that makes sense to me. It wasn't clear to me what was the bar for entry

Reply:
We don't have any defined bar for entry and I'm happy to add this feature if you think it makes the code more readable. I just generally find `getattr`/`hasattr` to be more descriptive if applicable.